pandas/core/arrays/_arrow_string_mixins.py

```diff
@@ -23,6 +26,13 @@ class ArrowStringArrayMixin:
     def __init__(self, *args, **kwargs) -> None:
         raise NotImplementedError
 
+    def _convert_int_dtype(self, result):
+        # Convert an int-dtype arrow result to an appropriate output type.
+        raise NotImplementedError
+
+    def _apply_elementwise(self, func: Callable) -> list[list[Any]]:
```

Comment:
This needs to be implemented for ArrowStringArray as well then?

Reply:
this is only used for the ArrowEA version.  The ArrowStringArray goes through _str_map, which ArrowEA doesn't have.  eventually id like to align the names, but there are too many branches/PRs as it is.

Reply:
Not sure what I am missing, but `_apply_elementwise` is called from the now-shared `_str_find` method just below, and so I would think that you can also get there from `ArrowStringArray._str_find` ?

Reply:
yep my bad.  ArrowStringArray inherits ArrowEA so gets its apply_elementwise from there.  putting it here just prevents mypy from complaining